### PR TITLE
Set up database

### PR DESCRIPTION
**Pull request by NikitaMasalkov**

### CODE REVIEW

**Comment by NikitaMasalkov** [comment]
image

File: app/controllers/dashboard_controller.rb [hunk 1]
```diff
@@ -1,3 +1,12 @@
 class DashboardController < ApplicationController
-  def index; end
-end
\ No newline at end of file
+  def index
+    tasks = Task.all.includes(:users)
+    tasks_waiting_list = tasks.select { |task| task.status == 'waiting_list' }
+    tasks_in_progress = tasks.select { |task| task.status == 'in_progress' }
+    tasks_done = tasks.select { |task| task.status == 'done' }
```
Comment (NikitaMasalkov) [inline app/controllers/dashboard_controller.rb]: convert to enum methods

File: app/javascript/stylesheets/dashboard.scss [hunk 2]
```diff
@@ -1,7 +1,3 @@
-.content {
-  padding-top: 80px;
+.p-t-3 {
```
Comment (NikitaMasalkov) [inline app/javascript/stylesheets/dashboard.scss]: no necessary, use pt-5

File: app/javascript/stylesheets/devise_alerts.scss [hunk 3]
```diff
@@ -0,0 +1,3 @@
+.p-t-4 {
```
Comment (NikitaMasalkov) [inline app/javascript/stylesheets/devise_alerts.scss]: use pt-6

File: app/javascript/stylesheets/devise_alerts.scss [hunk 4]
```diff
@@ -0,0 +1,3 @@
...
+  padding-top: 4rem;
```
Comment (NikitaMasalkov) [inline app/javascript/stylesheets/devise_alerts.scss]: !important

File: app/javascript/stylesheets/style.scss [hunk 5]
```diff
@@ -1 +1,6 @@
-@import 'dashboard';
\ No newline at end of file
+@import 'dashboard';
+@import 'devise_alerts';
+
+.container-pt-5r {
```
Comment (NikitaMasalkov) [inline app/javascript/stylesheets/style.scss]: proper name

File: app/models/attachment.rb [hunk 6]
```diff
@@ -0,0 +1,4 @@
+class Attachment < ApplicationRecord
+  belongs_to :user, optional: true
```
Comment (NikitaMasalkov) [inline app/models/attachment.rb]: is it necessary?

File: app/views/dashboard/index.html.erb [hunk 7]
```diff
@@ -1,58 +1,78 @@
...
-         Task "One morning, when Gregor Samsa woke from troubled dreams, he found himself transformed in his bed into a horrible vermin." assigned to @Petya
-      </li>
-      <li class="list-group-item">
-        Task "He lay on his armour-like back, and if he lifted his head a little he could see his brown belly, slightly domed and divided by arches into stiff sections." assigned to @Vasya
-      </li>
+  <div class="row">
+    <div class="col-md-4">
+      <ul class="list-group">
+        <% tasks_waiting_list.each do |task_waiting_list|  %>
+          <li class="list-group-item"><%= task_waiting_list.title.capitalize %></li>
```
Comment (NikitaMasalkov) [inline app/views/dashboard/index.html.erb]: bootstrap class for capitalize

File: app/views/dashboard/index.html.erb [hunk 8]
```diff
@@ -1,58 +1,78 @@
...
+
+<div class="container-fluid p-t-3">
+  <ul class="list-group">
+    <% tasks.each do |task| %>
       <li class="list-group-item">
-        Task "The bedding was hardly able to cover it and seemed ready to slide off any moment." assigned to @Katya
+        Task <a href="#"><%= task.title %></a>
+        assigned to:
+        <% task.users.each do |user| %>
+          <a href="#"><%= user.username %></a>
```
Comment (NikitaMasalkov) [inline app/views/dashboard/index.html.erb]: link_to

File: Gemfile [hunk 9]
```diff
@@ -11,7 +11,10 @@ gem 'pg', '>= 0.18', '< 2.0'
 gem 'puma', '~> 3.11'
 # Use SCSS for stylesheets
 gem 'sass-rails', '~> 5.0'
+# Custom gems
 gem 'webpacker'
+gem 'devise'
+gem 'pry'
```
Comment (vkalach) [inline Gemfile]: move to necessary section

File: app/javascript/stylesheets/style.scss [hunk 10]
```diff
@@ -1 +1,6 @@
-@import 'dashboard';
\ No newline at end of file
+@import 'dashboard';
+@import 'devise_alerts';
+
+.content-top-padding {
```
Comment (vkalach) [inline app/javascript/stylesheets/style.scss]: change class name

**Pull request merged**